src/gui/screen_printing.cpp

```diff
@@ -60,6 +60,7 @@ void screen_printing_data_t::tuneAction() {
     case printing_state_t::PRINTING:
     case printing_state_t::PAUSED:
         Screens::Access()->Open(GetScreenMenuTune);
+        change_print_state();
```

Comment:
GUI supports only one screen at the time.
So `Screens::Access()->Open(GetScreenMenuTune);` will close (destroy) Print screen.
But `change_print_state()` could cause extra unwanted redraw just before it is closed.

Please remove it.